### DISCUSSION TITLE
empty lists after upgrade

### DISCUSSION THREAD

**Discussion by ivang76** [statamic-rad-pack/runway#417] [Help] [answered]
Hello,

After the latest Runway upgrade, the list of items in the control panel linked to the models appears as empty rows. I followed the upgrade guide, created new resources, checked the field names and types, but still, nothing is being displayed.

<img width="1272" alt="image" src="https://github.com/statamic-rad-pack/runway/assets/7498876/bb8bc537-cbb9-47a3-b3f3-32f9d12c2150">

Any ideas?
thanks!

**Comment by duncanmcclean** [comment]
Did you manage to get this figured out?

**Reply by ivang76** [reply, marked as answer]
Yes, I had to remove the composer lock file and force another update. For some reason, updating only Runway was not sufficient.
